README.md

```diff
@@ -97,11 +103,154 @@ Please see Hazelcast Node.js [code samples](https://github.com/hazelcast/hazelca
 
 You can also refer to Hazelcast Node.js [API Documentation](http://hazelcast.github.io/hazelcast-nodejs-client/api/current/docs/).
 
-# Configuration
+# Configuration Overview
+
+## 1. Configuration Options
 
 You can configure Hazelcast Node.js Client declaratively (JSON) or programmatically (API).
 
-See [CONFIG.md](CONFIG.md) for details.
+* Programmatic configuration
+* Declarative configuration (JSON file)
+
+### 1.1. Programmatic Configuration
+
+For programmatic configuration of the Hazelcast Java Client, just instantiate a ClientConfig object and configure the
```

Comment:
...of the Hazelcast Node.js Client, just instantiate a `ClientConfig` object and...